### DISCUSSION TITLE
[ FF7 ] Name and HP boxes in battle are off with dialogue transparency on

### DISCUSSION THREAD

**Discussion by Rumbah2** [julianxhokaxhiu/FFNx#167] [Q&A]
With the latest canary 1.7.2.104 the battle scene does not seem to extend behind the Name and HP boxes.
That causes dark boxes as the HP and name boxes are against a black background and not the 3d scene.

I attached screenshots of different transparency and here for better comparison:
http://www.framecompare.com/image-compare/screenshotcomparison/77G6GNNX

I'm on the German Steam version of the game.

![transparency0](https://user-images.githubusercontent.com/40009325/84596900-9817a000-ae60-11ea-86c0-01594d9e2bc9.png)
![transparency25](https://user-images.githubusercontent.com/40009325/84596895-964ddc80-ae60-11ea-8d68-51ebbdd0b981.png)
![transparency50](https://user-images.githubusercontent.com/40009325/84596897-96e67300-ae60-11ea-9340-5e58e779f965.png)
![transparency75](https://user-images.githubusercontent.com/40009325/84596898-977f0980-ae60-11ea-9306-9a52183b7cf2.png)
![transparency100](https://user-images.githubusercontent.com/40009325/84596899-977f0980-ae60-11ea-870a-17ad2b3c02f3.png)

**Comment by julianxhokaxhiu** [comment]
Hey, thanks for the report! Did you try using the new hext files in the canary? Battle scenes should be stretched vertically now.

![image](https://user-images.githubusercontent.com/1237070/84603998-3c1a3f00-ae93-11ea-9d00-635e0cd60a36.png)

**Comment by Rumbah2** [comment]
Ok, I deleted my hext and shader folder and tried again with 1.7.2.106 and now it works. Seems to have been an old hext file then?

Anyway, works in the latest canary so I'll close this issue.
